### ISSUE TITLE
kmer model conversion

### ISSUE THREAD

**Issue by guangzhaocs** [nanoporetech/remora#194]
Hi,

I find two kmer models for RNA002 in another repository [kmer model](https://github.com/nanoporetech/kmer_models). The first is in standard units [5mer_levels_v1.txt](https://github.com/nanoporetech/kmer_models/blob/master/rna_r9.4_180mv_70bps/5mer_levels_v1.txt). While the seconde is the pA level [template_median69pA.model](https://github.com/nanoporetech/kmer_models/blob/master/legacy/legacy_r9.4_180mv_70bps_5mer_RNA/template_median69pA.model) in the legacy. My question is:

1. Do the two versions represent the same for rna_r9.4_180mv_70bps_5mer_levels?
2. How to convert the pA level to the standard units? Could you provide more detailed information for the conversion?

Thanks very much!

**Comment by marcus1487** [comment]
Yes. These models represent the expected levels for the same condition. You can find the method to convert to standard units [here](https://github.com/nanoporetech/remora/blob/v3.3.0/src/remora/refine_signal_map.py#L340).
